src/libs/Localize/LocaleListener/index.js

```diff
@@ -0,0 +1,11 @@
+import BaseLocaleListener from './BaseLocaleListener';
```

Comment:
Similar comment to the ones in `src/libs/Localize/LocaleListener/index.desktop.js`, I think this file could simply be:

```js
import BaseLocaleListener from './BaseLocaleListener';

export default {
    connect: BaseLocaleListener.connect,
};
```